genai-perf/tests/test_csv_exporter.py

```diff
@@ -228,3 +228,133 @@ def test_embedding_csv_output(
         ]
         returned_data = [data for _, data in mock_read_write]
         assert returned_data == expected_content
+
```

Comment:
Since these test cases are just testing goodput, can you just check the goodput-specific lines you expect are in the output? You don't need the other checks. They are already in the general purpose unit tests.